src/preact/base-element.js

```diff
@@ -259,17 +281,28 @@ PreactBaseElement['className'] = '';
 PreactBaseElement['passthrough'] = false;
 
 /**
- * Handling children with passthroughNonEmpty mode is the same as passthrough
+ * Handling children with passthroughNonEmpty mode is similar to passthrough
  * mode except that when there are no children elements, the returned
  * prop['children'] will be null instead of the unnamed <slot>.  This allows
  * the Preact environment to have conditional behavior depending on whether
- * or not there are children.  Consider using a Mutation Observer in your
- * component for detailed control of rerender when children are updated.
+ * or not there are children.
  *
  * @protected {boolean}
  */
 PreactBaseElement['passthroughNonEmpty'] = false;
 
+/**
+ * A static map for storing sets of options to be used in the Mutation Observer
+ *
+ * @protected {Object<string, Object<string, boolean|Array<string>>>}
+ */
+PreactBaseElement['mutationObserverOptions'] = {
```

Comment:
No need to hoist it on the constructor. It's just a basic module const. Somewhere up top: const mutationObserverOptions = {...}.

Reply:
Sounds good, thank you!